### PR TITLE
feat: add Renewal columns to finder table

### LINKED ISSUE #3200: Add Renewal related columns to finder table
https://www.figma.com/file/GGd3IzUsi6INf7n6BHdjdX/Org-Tabs?node-id=543%3A129278&mode=dev

### PR DESCRIPTION
**Pull request by acalinica**
## Proposed changes

closes #3200

## Changes

What types of changes does your code introduce?  _Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Build related changes
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Other (please describe below)

### CODE REVIEW

File: packages/apps/spaces/app/organizations/[id]/components/Tabs/panels/AboutPanel/AboutPanel.tsx [hunk 1]
```diff
@@ -194,6 +193,8 @@ export const AboutPanel = () => {
     },
   });
 
+  console.log(defaultValues);
```
Comment (Kasia9090) [inline packages/apps/spaces/app/organizations/[id]/components/Tabs/panels/AboutPanel/AboutPanel.tsx]: 💅 please remove console.log

File: packages/apps/spaces/app/organizations/[id]/components/Tabs/shared/FormSocialInput/FormSocialInput.tsx [hunk 2]
```diff
@@ -40,6 +40,7 @@ export const FormSocialInput = ({
 }: FormSocialInputProps) => {
   const { getInputProps } = useField(name, formId);
   const { value, onChange, onBlur } = getInputProps();
+  console.log('value', value);
```
Comment (Kasia9090) [inline packages/apps/spaces/app/organizations/[id]/components/Tabs/shared/FormSocialInput/FormSocialInput.tsx]: Same as above

File: packages/apps/spaces/app/organizations/[id]/components/Tabs/shared/FormSocialInput/FormSocialInput.tsx [hunk 3]
```diff
@@ -175,7 +176,7 @@ export const FormSocialInput = ({
 
   return (
     <>
-      {(values as Value[])?.map(({ id, url }, index) => (
```
Comment (Kasia9090) [inline packages/apps/spaces/app/organizations/[id]/components/Tabs/shared/FormSocialInput/FormSocialInput.tsx:178]: Let's name types as entities they are describing, eg `SocialInputValue`. That makes the code easier to read and understand what is happening especially after a while

File: packages/apps/spaces/components/organization/organization-details/renewal/RenewalForecastCell.tsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  potentialAmount = null,
+}: RenewalForecastCellProps) => {
+  return (
+    <Flex flexDir='column' justify='center'>
+      <Text fontSize='sm' color={amount ? 'gray.700' : 'gray.500'}>
+        {amount ? formatCurrency(amount) : 'Unknown'}
+      </Text>
+      {potentialAmount && potentialAmount !== amount && (
+        <Text fontSize='sm' color='gray.500' textDecoration='line-through'>
+          {formatCurrency(potentialAmount)}
```
Comment (Kasia9090) [inline packages/apps/spaces/components/organization/organization-details/renewal/RenewalForecastCell.tsx:22]: You could use here the helper function from utils that helps with elegant formatting of large numbers

**Review by Kasia9090 — CHANGES_REQUESTED** [review]

File: packages/apps/spaces/components/organization/organization-details/renewal/RenewalForecastCell.tsx (hunk 4 above)
Comment (acalinica) [reply packages/apps/spaces/components/organization/organization-details/renewal/RenewalForecastCell.tsx:22]: How is the function called? Or maybe you can give me the path to it.

**Review by Kasia9090 — APPROVED** [review]

**Pull request merged**